wcvendors/wcvendors
# Fixed: Bulk assign media to vendor #565

### All Submissions:






### Changes proposed in this Pull Request:
This PR adds new class `WCVendors_Admin_Media` to handle the media assignment. It adds a bulk action to the media admin page named Assign Vendor (using `wcv_get_vendor_name()`). When the admin choose Assign Vendor action, a select2 box will appear so the admin can choose which vendor to assign media files to.

### Knows issues:
* The bulk action work only in the list mode. Media page is set to grid mode by default.



Closes #565.

### How to test the changes in this Pull Request:

1. Go to Dashboard > Media
2. Select some image.
3. Choose Assign Vendor from the bulk actions select box.
4. See the new vendor select2 box appear.
5. Choose a vendor.
6. Click Apply button.
7. See the images now is set to the choosen vendor.
8. See a notice with information about vendor and number of processed images.
